### DISCUSSION TITLE
Unable to get the logs from my application through Opentelemetery Collector configuration to Apache Skywalking

### DISCUSSION THREAD

**Discussion by lijie123bes** [apache/skywalking#12434] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

Java Agent (apache/skywalking-java)

### What happened

warn zapgrpc/zapgrpc.go:195 [core] [Channel #1 SubChannel #2] grpc: addrConn.createTransport failed to connect to {Addr: "localhost:11800", ServerName: "localhost:11800", }. Err: connection error: desc = "transport: Error while dialing: dial tcp [::1]:11800: connectex: No connection could be made because the target machine actively refused it." {"grpc_log": true}

### What you expected to happen

normal

### How to reproduce

receivers: otlp: protocols: grpc:

processors: batch: exporters: otlp: endpoint: "localhost:11800" tls: insecure: true logging: loglevel: debug

service: pipelines: metrics: receivers: [otlp] processors: [batch] exporters: [logging,otlp]

This is my skywalking configuration file

receiver-sharing-server: selector: ${SW_RECEIVER_SHARING_SERVER:default} default:

### Are you willing to submit a pull request to fix on your own?

- [X] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
Can't see much information. Which OAP version are you using?

Here is docs for latest release, https://skywalking.apache.org/docs/main/latest/en/setup/backend/log-otlp/
